pre_commit_dbt/utils.py

```diff
@@ -176,7 +176,22 @@ def get_macros(
         split_key = key.split(".")
         filename = split_key[-1]
         if filename in filenames and split_key[0] == "macro":
-            yield Macro(key, macro.get("name"), filename, macro)  # pragma: no mutate
+            macro_ref = Macro(key, macro.get("name"), filename, macro)  # pragma: no mutate
+            yield macro_ref
+
+
+def get_macro_name(macro_ref: str) -> str:
+    return macro_ref.split(".")[-1]
+
+
+def get_macros_names_in_models(manifest: Dict[str, Any]) -> str:
```

Comment:
Is the output of the function really a string?

Reply:
It's a type `List[str]`